HOW_TO.md

```diff
@@ -1,21 +1,21 @@
 # How To Use This Template
```

Comment:
This needs to be integrated with the README and the original version of this file reverted. Look at other transformer's README for an examples as what to add, as well as what you updated for the other transformer.